README.adoc

```diff
@@ -19,15 +19,22 @@ List of arguments below.
 == Command line arguments:
 
 - [0] - name of the first player
-- [1] - sign of the first player ("O" or "o" for Naught and "X" or "x" for Cross - second player will get remaining sign)
+- [1] - sign of the first player ("O" or "o" for Naught and "X" or "x" for Cross - second player will get the remaining sign)
 - [2] - name of the second player
-- [3] - dimension of board (in range from 3 to 30)
+- [3] - dimension of board (in range from 3 to 30). Board is a square.
 - [4] - winning rule - how many field must filled with one type of sign to win (range from 3 to board dimension - 1 (if greater than 3))
-- [5] - language tag: "en" for english (default language), "pl" - polish, "ja" - japanese (not complete by now)
+- [5] - language tag: "en" for english (default language), "pl" - polish, "ja" - japanese (not completed by now)
 
 Example of command:
 `java -jar Aleksander O Czesio 5 3 pl`
 Command for playing as Aleksander (naught) and Czesio (cross), board will be 5x5, winning rule 3 and language will be set to polish.
 
+== Python script for automated end-to-end tests
 
-image::crc.png[]
```

Comment:
Wycięto obrazek? Wolałbym mieć gdzieś obrazek, np. z dopiskiem że nieaktualny. :(